cmd/minikube/cmd/start.go

```diff
@@ -811,6 +812,17 @@ func validateFlags(cmd *cobra.Command, drvName string) {
 		}
 	}
 
+	if cmd.Flags().Changed(containerRuntime) {
+
+		runtime := viper.GetString(containerRuntime)
+
+		_, isValidRuntime := util.ValidateString(runtime, cruntime.ValidOptions())
```

Comment:
change this to a simple check, no need for util.
make sure to change the user input to lower case,
so if user enters Docker it should be converted and saved as "docker" or Containerd to be saved as containerd